src/applications/mhv-landing-page/selectors/personalInformation.js

```diff
@@ -1,17 +1,7 @@
-import _ from 'lodash';
-
 export const selectGreetingName = state => {
-  const result =
-    state?.myHealth?.personalInformation?.data?.preferredName ||
+  return (
+    state?.user?.profile?.preferredName ||
     state?.user?.profile?.userFullName?.first ||
-    null;
-
-  if (!!result && result === result.toUpperCase()) {
-    return _.capitalize(result);
-  }
-
-  return result;
```

Comment:
Removed this as there's another ticket to [Preserve/display user's capitalization for Preferred Name](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97412).